edelphi/src/main/resources/fi/metatavu/edelphi/liquibase/query-template-settings.sql

```diff
@@ -0,0 +1,190 @@
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'scale2d') from QueryPageSettingKey where name = 'thesis.showLiveReport';
+INSERT INTO QueryPageTemplateSimpleSetting (id, value) select (select max(id) from QueryPageTemplateSetting), '0';
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'timeserie') from QueryPageSettingKey where name = 'thesis.commentable';
+INSERT INTO QueryPageTemplateSimpleSetting (id, value) select (select max(id) from QueryPageTemplateSetting), '1';
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'timeserie') from QueryPageSettingKey where name = 'thesis.showLiveReport';
+INSERT INTO QueryPageTemplateSimpleSetting (id, value) select (select max(id) from QueryPageTemplateSetting), '0';
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'timeline') from QueryPageSettingKey where name = 'thesis.commentable';
+INSERT INTO QueryPageTemplateSimpleSetting (id, value) select (select max(id) from QueryPageTemplateSetting), '1';
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'timeline') from QueryPageSettingKey where name = 'thesis.showLiveReport';
+INSERT INTO QueryPageTemplateSimpleSetting (id, value) select (select max(id) from QueryPageTemplateSetting), '0';
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'grouping') from QueryPageSettingKey where name = 'thesis.commentable'; 
+INSERT INTO QueryPageTemplateSimpleSetting (id, value) select (select max(id) from QueryPageTemplateSetting), '1';
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'grouping') from QueryPageSettingKey where name = 'thesis.showLiveReport';
+INSERT INTO QueryPageTemplateSimpleSetting (id, value) select (select max(id) from QueryPageTemplateSetting), '0';
+INSERT INTO QueryPageTemplateSetting (id, key_id, queryPageTemplate_id) select (select max(id) + 1 from QueryPageTemplateSetting), id, (select id from QueryPageTemplate where iconName = 'multiselect') from QueryPageSettingKey where name = 'thesis.commentable';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "select" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "select" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The line length is greater than 80 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3ALineLength)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "from" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "select" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "from" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "where" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "from" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "where" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal 'multiselect' 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)